File: java/01_EinUndAusgabe/Print.java
public class Print {
    public static void main(String[] args) {
        // Ausgabe mit Zeilenvorschub
        System.out.println("Ausgabe mit Zeilenvorschub");
        System.out.println("Ausgabe mit Zeilenvorschub");
        System.out.println("Ausgabe mit Zeilenvorschub");

        // Ausgabe ohne Zeilenvorschub
        System.out.print("Ausgabe ohne Zeilenvorschub");
        System.out.print("Ausgabe ohne Zeilenvorschub");
        System.out.print("Ausgabe ohne Zeilenvorschub");
        System.out.println();

        // Ausgabe mit Escapezeichen (% bzw \)
        System.out.printf("Zeichen: %c, Zeichenkette: %s, Int: %d\n", 'a', "Xyz", 45054);
        System.out.printf("Int (Hex): %h\n", 45054);
        System.out.println();

        System.out.printf("Zeichenkette (Groß): '%S'\n", "das ist mal ein text");
        System.out.printf("Zeichenkette (mindestgröße/rechtsbündig): '%10s'\n", "abcde");
        System.out.printf("Zeichenkette (mindestgröße/rechtsbündig): '%10s'\n", "abcdeghijklmnopqrstuvwxyz");
        System.out.printf("Zeichenkette (linksbündig): '%-10s'\n", "abcde");
        System.out.println();

        System.out.printf("Float/Double: %f\n", 45054.292929);
        System.out.printf("Float/Double: %7.2f\n", 45054.292929);
    }
}

File: java/01_EinUndAusgabe/Print.py
print("Ausgabe mit Zeilenvorschub")
print("Ausgabe mit Zeilenvorschub")
print("Ausgabe mit Zeilenvorschub")

print("Ausgabe ohne Zeilenvorschub", end="")
print("Ausgabe ohne Zeilenvorschub", end="")
print("Ausgabe ohne Zeilenvorschub", end="")
print()

print("Zeichen: {}, Zeichenkette: {}, Int: {}".format("a", "Xyz", 45054))
print("Int (Hex):{}".format(hex(45054)))
print()

print("Zeichenkette (Groß): '{}'".format("das ist mal ein text".upper()))
print("Zeichenkette (mindestgröße/rechtsbündig): '{:>10}'".format("abcde"))
print("Zeichenkette (mindestgröße/rechtsbündig): '{:>10}'".format("abcdeghijklmnopqrstuvwxyz"))
print("Zeichenkette (linksbündig): '{:<10}'".format("abcde"))

print("Mehrere Parameter: {0} {1} {2}".format("Null", "Eins", "Zwei"))
print("Mehrfache Angabe eines Parameters: {0} {1} {2} {2} {2} {1} {0}".format("Null", "Eins", "Zwei"))
print("Benannte Parameter: {vorname} {nachname}".format("Max", "Mustermann"))
print()

print("Float/Double: {:f}".format(45054.292929))
print("Float/Double: {:7.2f}".format(45054.292929))
